lib/core/src/Cardano/Wallet/Gen.hs

```diff
@@ -225,19 +225,32 @@ shrinkByteString bs
   where
     n = BS.length bs
 
-genText :: Gen Text
-genText =
-    (T.pack . take 32 . getPrintableString <$> arbitrary) `suchThat` guardText
+genTxMetaText :: Gen Text
+genTxMetaText =
+    (T.pack . take 32 . getPrintableString <$> arbitrary)
```

Comment:
We could make this variable-length, with a hard maximum of 64 unicode characters, which is then filtered by `guardTxMetaText`.

```suggestion
    (T.pack . getUnicodeString <$> scale (min 64) arbitrary)
```

Also I believe it's fine to use `UnicodeString` instead of `PrintableString`.

Reply:
Fixed in 58cc2af86cf7d3029d4676963acf065d7b63e18b.